### ISSUE TITLE
cannot run hg38_hyena_seqlen_warmup_reload

### ISSUE THREAD

**Issue by sustcsonglin** [HazyResearch/hyena-dna#45]
Hi,

Thanks for your great work! I am trying to do `hg38/hg38_hyena_seqlen_warmup_reload.yaml` experiment. Got the following error msg:

<img width="685" alt="image" src="https://github.com/HazyResearch/hyena-dna/assets/30831390/b94612ab-5c8b-4d6c-b19b-a87f29f8db31">

I had some initial search on this issue and found [this](https://github.com/HazyResearch/hyena-dna/issues/31). I set `monitor: test/loss` and it still doesn't work. But i have no problem running 'g38/hg38_hyena.yaml'.  

Do you have any insights on this issue? Is this related to the sentence length warmup callback? because I can run 'g38/hg38_hyena.yaml' without this callback. i am using pytorch_lightning v1.8.6

**Comment by sustcsonglin** [comment]
After some inspections, I found the reason! [link1](https://github.com/HazyResearch/hyena-dna/blob/main/configs/experiment/hg38/hg38_hyena_seqlen_warmup_reload.yaml#L100C1-L100C36) disables loading the test dataloader during training, but the monitor metric was not changed. Hope this configuration will be fixed

**Comment by sustcsonglin** [comment]
Another question: does this configuration file consider the `trainer.devices` for setting `accumulate_grad_batches`? I didn't find sth like dividing by `trainer.devices` [here](https://github.com/HazyResearch/hyena-dna/blob/main/train.py#L632C1-L638C1)
